client/packages/common/src/ui/layout/tables/components/Cells/RadioCell.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { CellProps, Radio, RecordWithId } from '@openmsupply-client/common';
+
+export const RadioCell = <T extends RecordWithId>({
```

Comment:
```suggestion
export const SelectionCell = <T extends RecordWithId>({
```
Feels like this is a bit more specialised than just a radio? Maybe just me?

Reply:
Radio is already single selection only 👀  but can rename... would have to be like SingleSelectionCell tho...